lte/cloud/go/lte/const.go

```diff
@@ -18,23 +18,24 @@ limitations under the License.
 	are reproduced and annotated below.
 	Note: starred Swagger models also have mutable versions of the model.
 
-	Configurator type     Swagger model         Edges out                                  Notes
-	-----------------     -------------         ---------                                  -----
-	apn                   apn                                                              Stored as apn_configuration
-	apn_policy_profile    apn_policy_profile    apn,policy                                 Internal-only
-	apn_resource          apn_resource          apn
-	base_name             base_name_record      policy
-	cellular_enodeb       enodeb                                                           Stored as enodeb_configuration
-	cellular_gateway      *lte_gateway          cellular_enodeb,apn_resource               Stored as gateway_cellular_configs
-	policy                policy_rule           policy_qos_profile                         Stored as policy_rule_config
+	Configurator type     Swagger model          Edges out                                  Notes
+	-----------------     -------------          ---------                                  -----
+	apn                   apn                                                               Stored as apn_configuration
+	apn_policy_profile    apn_policy_profile     apn,policy                                 Internal-only
+	apn_resource          apn_resource           apn
+	base_name             base_name_record       policy
+	cellular_enodeb       enodeb                                                            Stored as enodeb_configuration
+	cellular_gateway      *lte_gateway           cellular_enodeb,apn_resource               Stored as gateway_cellular_configs
+	cellular_gateway_pool *cellular_gateway_pool cellular_gateway                           Stored as cellular_gateway_pool_configs
+	policy                policy_rule            policy_qos_profile                         Stored as policy_rule_config
 	policy_qos_profile    policy_qos_profile
 	rating_group          *rating_group
 	subscriber            *subscriber           apn,policy,base_name,apn_policy_profile
 
 	Resulting DAG
 
-	cellular_gateway -.-> cellular_enodeb
-	                  '-> apn_resource -> apn*
+	cellular_gateway_pool -.-> cellular_gateway -.-> cellular_enodeb
```

Comment:
Only small thing is that first relation can just use the short arrow, i.e. `cellular_gateway_pool -> cellular_gateway ...`